src/common/utils/pathUtils.ts

```diff
@@ -1,5 +1,24 @@
+import { Uri } from 'vscode';
 import { isWindows } from '../../managers/common/utils';
 
+export function checkUri(scope?: Uri | Uri[] | string): Uri | Uri[] | string | undefined {
+    if (scope instanceof Uri) {
+        if (scope.scheme === 'vscode-notebook-cell') {
+            return Uri.from({
```

Comment:
@eleanorjboyd  this isn't a valid notebook uri
 haven't looked at the code, perhaps this has some meaning in this extension
However of you want to find the notebook Uri, you must iterate through all notebooks and find the notebook that has a cell with the matching Uri .